src/components/input/demo/index.tsx

```diff
@@ -309,11 +309,44 @@ export class SmoothlyInputDemo {
 					</div>
 				</smoothly-form>
 				<h4>Smoothly Radio Buttons</h4>
-				<smoothly-form onSmoothlyFormSubmit={e => console.log(e.detail)}>
-					<smoothly-input-radio name="radioNew">
-						<smoothly-input-radio-button value={"first"}>Label 1</smoothly-input-radio-button>
-						<smoothly-input-radio-button value={"second"}>Label 2</smoothly-input-radio-button>
-						<smoothly-input-radio-button value={"third"}>Label 3</smoothly-input-radio-button>
+				<smoothly-form onSmoothlyFormSubmit={e => console.log(e.detail)} looks="border">
+					<smoothly-input-radio clearable name="radioFirstInput">
+						<smoothly-icon name="checkmark-circle" slot="start" />
+						<smoothly-input-clear size="icon" slot="end">
+							<smoothly-icon name="close" />
+						</smoothly-input-clear>
+						<p slot="label">Clearable</p>
+						<smoothly-input-radio-item slot="options" value={"first"}>
+							Label 1
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item selected slot="options" value={"second"}>
+							Label 2
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item slot="options" value={"third"}>
+							Label 3
+						</smoothly-input-radio-item>
+					</smoothly-input-radio>
+					<smoothly-input-radio name="radioSecondInput">
+						<p slot="label">Not clearable</p>
+						<smoothly-input-radio-item slot="options" value={"first"}>
+							Label 1
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item slot="options" value={"second"}>
+							Label 2
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item slot="options" value={"third"}>
+							Label 3
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item slot="options" value={"third"}>
+							Label 3
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item slot="options" value={"third"}>
+							Label 3
+						</smoothly-input-radio-item>
+						<smoothly-input-radio-item slot="options" value={"third"}>
+							Label 3
+						</smoothly-input-radio-item>
+
```

Comment:
linter warning. run `npm run fix` to fix. (view this comment in the file and you should see the warning)